### ISSUE TITLE
Font - Default Font Family

### ISSUE THREAD

**Issue by bryphe** [revery-ui/revery#143]
We force the user at the moment to pick a font via `fontFamily`. If a `fontFamily` is not specified, you get a crash with this exception: 
```
- Loading font:
Fatal error: exception Fontkit.FontKitLoadFaceException("[ERROR]: Unable to load font at FT_New_Face\n")
```

This is pretty rough - we should pick out a default font family if none is specified. I think that the `Roboto-Regular.ttf` that we already bundle, and use in several examples, would be an OK default. 

Does anyone have any thoughts / pereferences on this?

**Comment by jchavarri** [comment]
My only suggestion would be around payload size in the jsoo builds, in order to try to keep the web bundles small in the future. We could pick a custom font with a small subset of glyphs, or even better, choose a font that is available in browsers by default. In the last case, we could make it so `reason-fontkit` doesn't even try to load anything because it knows that function is already provided by the browser 🤔

**Comment by jchavarri** [comment]
Maybe related: https://parall.ax/blog/view/3072/tutorial-reducing-the-file-size-of-custom-web-fonts

**Comment by baransu** [comment]
More up to date error when no `fontFamily` is provided in `Text` style:
```
Fatal error: exception (Invalid_argument "\"\": invalid path")
Raised at file "stdlib.ml", line 30, characters 20-45
Called from file "src/Path.re", line 20, characters 14-26
Called from file "src/Core/Environment.re", line 87, characters 6-30
Called from file "src/Font/FontCache.re", line 29, characters 22-56
Called from file "src/Draw/Text.re", line 10, characters 9-37
Called from file "src/Draw/Text.re" (inlined), line 18, characters 16-59
Called from file "src/UI/TextNode.re", line 145, characters 20-66
Called from file "src/lib/LayoutSupport.re", line 1303, characters 10-85
Called from file "src/lib/Layout.re", line 281, characters 6-192
Called from file "src/lib/Layout.re", line 456, characters 8-264
Called from file "src/lib/Layout.re", line 759, characters 17-684
Called from file "src/lib/Layout.re", line 281, characters 6-192
Called from file "src/lib/Layout.re", line 456, characters 8-264
Called from file "src/lib/Layout.re", line 759, characters 17-684
Called from file "src/lib/Layout.re", line 281, characters 6-192
Called from file "src/lib/Layout.re", line 557, characters 6-198
Called from file "src/lib/Layout.re", line 1554, characters 12-205
Called from file "src/lib/Layout.re", line 281, characters 6-192
Called from file "src/lib/Layout.re", line 1613, characters 7-213
Called from file "src/UI/Render.re", line 51, characters 2-45
Called from file "src/Core/Window.re", line 293, characters 2-12
Called from file "list.ml", line 110, characters 12-15
Called from file "src/Core/App.re", line 226, characters 6-123
Called from file "src/sdl2.re", line 587, characters 10-20
```

**Comment by glennsl** [comment]
Fixed by #879
